packages/url_launcher/url_launcher_android/example/pubspec.yaml

```diff
@@ -28,3 +28,8 @@ dev_dependencies:
 
 flutter:
   uses-material-design: true
+
+# FOR TESTING AND INITIAL REVIEW ONLY. DO NOT MERGE.
+# See https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#changing-federated-plugins
+dependency_overrides:
```

Comment:
This should not be needed; the platform interface should be using version constraints at this point.